language/diem-framework/modules/AccountAdministrationScripts.move

```diff
@@ -0,0 +1,534 @@
+address 0x1 {
+/// This module holds transactions that can be used to administer accounts in the Diem Framework.
```

Comment:
Might help to add one or two examples of what administration means (e.g., key rotation).